SourceX/dx.cpp

```diff
@@ -205,6 +205,21 @@ void BltFast(DWORD dwX, DWORD dwY, LPRECT lpSrcRect)
 	bufferUpdated = true;
 }
 
+/**
+ * @brief Limit FPS to lower CPU when not waiting for V-Blank

```

Comment:
s/not waiting/waiting/?

Reply:
No, this function is to be used when *waiting for v-blank* (aka v-sync) is not avalible.